### DISCUSSION TITLE
Welcome to Smilodon Discussions!

### DISCUSSION THREAD

**Discussion by benrick** [DevChatter/Smilodon#30] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kirkbrauer** [comment]
👋 Hi everyone! My name is Kirk Brauer.

I am a fullstack software developer and senior undergraduate computer science student at The Ohio State University. Go Bucks!

I currently work at an eCommerce startup called Jatango, which develops software for online sellers who want to sell their products live on social media. In my job I primarily work with .NET, React, and TypeScript, however I also dabble in other things such as Kubernetes, Docker, Terraform, and Rust.

My areas of interest include: programming languages, software engineering, distributed systems, and safety-critical systems. My university research focuses on autonomous driving systems for self-driving vehicles and safety-critical software engineering. After graduating, I plan on pursuing a Master's of Science in Computer Science and Engineering with an emphasis on safety-critical systems.

I want to help this community build an even better implementation of Mastodon that takes full advantage of the performance and developer experience that come with .NET. I want to use this project as an opportunity to increase my contributions to OSS and the fediverse as a whole.

**Comment by KristofferStrube** [comment]
Hey, I'm Kristoffer Strube.

I'm a .NET developer from Aarhus, Denmark that recently finished my Master's in Computer Science. In my free time, I work a lot with Blazor and specifically JS Interop which includes JSON handling and reading W3C documents. My Master's thesis was on Data Intensive Systems and I have done a lot of projects that work with Knowledge Graphs which I think fits pretty well with the concepts that ActivityPub also focuses on.

I have worked a lot with SignalR, Blazor, SVG's and animations, and general data management.

I have started working on wrapping the core of ActivityPub which is ActivityStreams in another repo and I think this project could use that once published. https://github.com/KristofferStrube/ActivityStreams

I would love to contribute to this project in my free time.
